### DISCUSSION TITLE
Using the same tag of helm chart no update by argocd

### DISCUSSION THREAD

**Discussion by jlivatthana** [argoproj/argo-cd#19888] [Q&A]
ArgoCD : v2.11.7+e4a0246

Hello,
When we package et push a helm chart in our registry with the same tag in our registry, argocd dont pull the last helm chart. The sha in the registry change but not the tag. We need to use the same tag because we made a mistake in the first push. 
Do you know how to say argocd to pull again the new helm chart and dont keep the old one ?

**Comment by jannfis** [comment]
You can use a hard refresh of the application to have Argo CD re-pull and re-render a chart with the same version that is already synced to the cluster.

As an alternative, you can wait 24 hours for the repository cache to expire or edit the Helm values used to render the chart.

**Reply by jlivatthana** [reply]
Thanks It works after searching very well the "hard refresh" button on GUI. I could refresh it ( the button hard refresh was only accessible by click very precisely on the small arrow of the refresh button) If you miss the down arrow you cannot have this option.
